README.md

```diff
@@ -35,6 +35,9 @@ Supported props
 | touch | boolean | true | If touch gestures should be enabled |
 | touchHandleWidth | number | 20 | Width in pixels you can start dragging from the edge when the sidebar is closed. |
 | dragToggleDistance | number | 30 | Distance the sidebar has to be dragged before it will open/close after it is released. |
+| pullRight | boolean | false | Place the sidebar on the right |
+| topSidebar | number | 0 | Sidebar distance from the top. |
```

Comment:
React sidebar positions itself using a `div` that uses `position: absolute` to have it take over the whole area of the `div`. If you wrap React sidebar in another `div` that sets the `position` style attribute, it will not take over the whole screen but only the parent `div`. Therefore, I can't accept the `topSidebar` feature.